rest/tests/test_rest.py

```diff
@@ -4,31 +4,12 @@
 from __future__ import unicode_literals
```

Comment:
I've renamed it `test_misc` since it's intended for REST-related tests to go there if they don't fit elsewhere